### ISSUE TITLE
Variable cannot be passed to included file

### ISSUE THREAD

**Issue by Ircama** [pwwang/liquidpy#8]
Thanks for this code!

I think this is not working:

file_A:
```
{% include "file_B" foo="test" %}
```

file_B:
```
{{ include.foo }}
```

Variable cannot be passed to included file

**Comment by pwwang** [comment]
You don't have to pass values to the included file. It can access variables assigned before it's included in the current template.

`file_A`:
```liquid
{% assign foo = "test" %}
{% include "file_B" %}
```

`file_B`:
```liquid
{{foo}}
```

But this is a good way to protect some variables. I will think about it.

**Comment by Ircama** [comment]
Yes, the method you documented is the workaround I did.

**Comment by pwwang** [comment]
Implemented in `v0.5.0`

https://liquidpy.readthedocs.io/en/latest/extends_include/#inclusion
